llama_cpp_guidance/llm.py

```diff
@@ -82,42 +86,46 @@ def session(self, asynchronous=False):
             raise NotImplementedError
 
     def __call__(self, *args, **kwargs):
-        logger.debug("Invoking LlamaCpp ({args}) ({kwargs})", args=args, kwargs=kwargs)
-        output = self.llm(*args, **kwargs)
-        logger.debug(
-            "LlamaCpp generated text: {text} ({choices})",
-            text=output["choices"][0]["text"],
-            choices=[
-                (choice["text"], choice["logprobs"]) for choice in output["choices"]
-            ],
-        )
+        print("Invoking LlamaCpp ({args}) ({kwargs})", args, kwargs)
```

Comment:
Is there a specific reason why this line has been converted to a print statement instead of a debug log? Ideally, we can trust the calling code to decide whether to print logging from the library.

Reply:
ugh sorry, it was just easier for me to debug this way and I forgot to change it back...

Reply:
No worries!